idl/foxsi-smex-setup-script.pro

```diff
@@ -9,7 +9,8 @@ CD, Current=fs_location
 ; with foxsi_
 
 foxsi_root_path = fs_location
-foxsi_data_path = strmid(foxsi_root_path, 0, strlen(foxsi_root_path)-3) + 'data/'
+;foxsi_data_path = strmid(foxsi_root_path, 0, strlen(foxsi_root_path)-3) + 'data/'
+foxsi_data_path = foxsi_root_path + '/data/'
```

Comment:
@aringlis you need to revert this change.  The weird (unexplained) code is to strip the `idl/` from `foxsi_root_path` and replace it with `data/`.  Your version doesn't work because `data/` is not a subdirectory of `idl/`.